content/copy.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Adding GitHub App managers in your organization
+intro: 'Organization owners can grant users the ability to manage some or all {% data variables.product.prodname_github_apps %} owned by the organization.'
+redirect_from:
+  - /articles/adding-github-app-managers-in-your-organization
+  - /github/setting-up-and-managing-organizations-and-teams/adding-github-app-managers-in-your-organization
+versions:
+  fpt: '*'
+  ghes: '*'
+  ghae: '*'
+  ghec: '*'
+topics:
+  - Organizations
+  - Teams
+shortTitle: Add GitHub App managers
+---
+
+This is a test!!!!!
+
+For more information about {% data variables.product.prodname_github_app %} manager permissions, see "[Roles in an organization](/organizations/managing-peoples-access-to-your-organization-with-roles/roles-in-an-organization#github-app-managers)."
+
+## Giving someone the ability to manage all {% data variables.product.prodname_github_apps %} owned by the organization
+
+{% data reusables.profile.access_org %}
+{% data reusables.profile.org_settings %}
+{% data reusables.organizations.github-apps-settings-sidebar %}
+
+1. Under "Management", type the username of the person you want to designate as a {% data variables.product.prodname_github_app %} manager in the organization, and click **Grant**.
+
+    ![Add a {% data variables.product.prodname_github_app %} manager](/assets/images/help/organizations/add-github-app-manager.png)
+
```

Comment:
☝️ This warning shouldn't be happening, but the overall Vale result is still promising because the `perl` command in the workflow is successfully stripping out Liquid. If it were _not_ successfully stripping out Liquid, Vale would be reporting a lot more errors (complaining about stuff like `prodname_github_app` above).

So the fact that it's not reporting those means our Liquid removal is a success. 🎉 You can see more proof of that by looking below at `data/ghec-only.md`, where Vale is not reporting any errors.

So why the `png` report here? We determined there are some list spacing issues in this article that make it invalid CommonMark. Vale uses CommonMark for parsing Markdown, so what's happening is the parsing isn't working properly, and Vale ends up reporting spurious errors. (It would be helpful for debugging if Vale reported anywhere that parsing failed, but it just fails silently, so we have to draw our own conclusions.)

The good news is that if we fix the CommonMark issues across our docs, we _should_ get full success with Vale.